docs/MakeRelease.html

```diff
@@ -32,16 +32,26 @@ <h1>Creating a new release</h1>
 in all <code>cmake</code> commands (5 instances at the time of last update of this documentation).
 <li>Commit and push this change to trigger a new build on GitHub Actions.
 Wait for that build to complete.
-<li>Download the 7 artifacts. Unzip them and rename  them to the following:
+<li>Download the 5 artifacts. Unzip them and rename them to the following:
```

Comment:
Also the 18.04. Tar files are not portable across Ubuntu versions.

Reply:
Not sure I understand this. See my other response.